lib/rubyplot/artist/figure.rb

```diff
@@ -108,6 +108,9 @@ def write(file_name, device: :file)
         Rubyplot.backend.canvas_height = @height
         Rubyplot.backend.canvas_width = @width
         Rubyplot.backend.figure = self
+        if ENV["RUBYPLOT_BACKEND"] == "MAGICK"
```

Comment:
FInd a way to move this to the backend. I don't want anything backend specific in the front end.

If you feel a compulsive need to do something like this, we should change the whole design and not do such patch work. Such things dont scale.

Reply:
I will try to find a way to add it into the magick backend, one another way would be to add this function to GR backend as well.

Reply:
I want to merge this PR, however I cannot do it unless you remove the `if` here. Please resolve this on priority before starting work for this week.

Reply:
[[Commit Link]](https://github.com/SciRuby/rubyplot/pull/45/commits/f07c3a71aa77f8017b8b22d936571b12e4724c09)  
Done, shifted it to backend, the task is performed during init_output_device